oauth2client/django_orm.py

```diff
@@ -46,12 +46,16 @@ def to_python(self, value):
             return None
         if isinstance(value, oauth2client.client.Credentials):
             return value
-        return pickle.loads(base64.b64decode(value))
+        return pickle.loads(base64.b64decode(smart_bytes(value)))
 
-    def get_db_prep_value(self, value, connection, prepared=False):
+    def get_prep_value(self, value):
         if value is None:
             return None
-        return base64.b64encode(pickle.dumps(value))
+        return smart_text(base64.b64encode(pickle.dumps(value)))
+
+    def value_to_string(self, obj):
```

Comment:
RE: "BTW: Is it better to add additional commits to the PR or amend current commit with the changes?"

@nathanielmanistaatgoogle Usually tells people to make new commits during review and then squash into a single commit before doing the merge